### ISSUE TITLE
调试grpo trainner 代码出现的问题

### ISSUE THREAD

**Issue by love112358** [NJUxlj/Travel-Agent-based-on-Qwen2-RLHF#5]
首先这个代码有一些问题 比如一些参数是错的 在源代码中没有补全 我对代码进行补全了 最后跑通代码出现下面的问题，loss': -1.1454590274735906e+30, 'grad_norm': nan, 'learning_rate': 1.927536231884058e-05, 'epoch': 0.11
这里的grad_norm': nan值是错的

<img width="781" height="216" alt="Image" src="https://github.com/user-attachments/assets/612ec0a4-59cc-457c-8e02-94332829f4a0" />
下面是对应的原代码部分信息：

<img width="841" height="246" alt="Image" src="https://github.com/user-attachments/assets/25f75e60-3e24-4d38-b6d2-7e9656715580" />

<img width="488" height="91" alt="Image" src="https://github.com/user-attachments/assets/7bb28e01-a4e2-4108-a050-b57b2863cecc" />

下面的代码是我自己修改的部分

<img width="946" height="338" alt="Image" src="https://github.com/user-attachments/assets/b18554fa-4185-4dab-920f-d9a9e21c7192" />

<img width="639" height="243" alt="Image" src="https://github.com/user-attachments/assets/7bb9d885-c497-4f9e-b0ec-e9f15b167d4c" />
原因是加载和处理验证集的数据集在通过_data_filter函数过滤时报错 验证集加载的数据集为 sft的数据集既：DATA_PATH = "src/data/travel_qa"
我不知道loss 和grad_norm 值的异常是不是和这个有关系，我使用的数据集为：train集为：
train_dataset = load_dataset(self.dataset_name_or_path, split='train').select(range(train_size)) 这里使用的：
DPO_DATA_PATH = "./src/data/Human-Like-DPO-Dataset"
eval_dataset = load_dataset(self.sft_dataset_name_or_path)  使用的是DATA_PATH = "src/data/travel_qa"
另外,
代码里面有好多参数错误：

<img width="365" height="212" alt="Image" src="https://github.com/user-attachments/assets/59c09955-5d4f-4499-8757-a527eb4f0da9" />

<img width="628" height="734" alt="Image" src="https://github.com/user-attachments/assets/66340778-8b9a-4795-b05c-32beb59d707c" /> 希望作者能提供一个正确的版本
